### PR TITLE
Prefill HCA `zipcode` or `postalCode` depending on country

### PR DESCRIPTION
**Pull request by rlbaker**
In the 10-10EZ json schema, if `country` is `"USA" | "MEX" | "CAN"`, then `zipcode` is used to populate the postal code. Otherwise, `postalCode` is used.

This PR allows pre-filling to match the expected key format for `zipcode`/`postalCode` depending on the country.

There may be a better way to solve this issue (possibly just making everything consistently `postalCode`), but this PR solves the issue described.

### CODE REVIEW

File: spec/models/form_profile_spec.rb [hunk 1]
```diff
@@ -368,6 +369,9 @@ def expect_prefilled(form_id)
       end
     end
 
+    context 'with a healthcare application form' do
+    end
```
Comment (lihanli) [inline spec/models/form_profile_spec.rb]: missing code or not needed?

Comment (rlbaker) [reply spec/models/form_profile_spec.rb]: Ah, leftover from an earlier spec I wrote to test this. I'll take it out.

File: app/models/form_profile.rb [hunk 2]
```diff
@@ -190,6 +193,13 @@ def initialize_contact_information(user)
     )
   end
 
+  # For 10-10ez forms, this function is overridden to provide a different
+  # key for postal_code is used depending on the country. The default behaviour
+  # here is used for other form types
+  def derive_postal_code(user)
+    { postal_code: user.va_profile.address.postal_code } if user.va_profile&.address
```
Comment (lihanli) [inline app/models/form_profile.rb]: `if user.va_profile&.address` is already checked in the last line

**Comment by rlbaker** [comment]
@jbalboni Do you mind weighing in on this one? It introduces what I might consider unnecessary complexity to deal with a simple difference in key name, but I wasn't sure whether we wanted to solve this in the schema and have to make changes in the frontend and backend form processing code as well (rather than just prefill).

**Comment by jbalboni** [comment]
@rlbaker I agree that the key difference is annoying (that's why the other forms use the same key all the time). But I think we need a solution for this problem soon-ish and changing the schemas is a longer process that we might not have time for right now with preneed going on.

**Pull request merged**